docs/quickstart.rst

```diff
@@ -60,38 +23,27 @@ users will start running within their own green gevent thread.
 
 .. code-block:: python
 
-    wait_time = between(5, 9)
-
-Our class defines a ``wait_time`` function that will make the simulated users wait between 5 and 9 seconds after each task 
-is executed. 
-
-.. code-block:: python
-
-    @task(2)
-    def index(self):
-        self.client.get("/")
-        self.client.get("/ajax-notifications/")
+    @task
+    def hello(self):
+        self.client.get("/hello")
+        self.client.get("/world")
     
-    @task(1)
-    def view_post(self):
+    @task(3)
+    def view_item(self):
+        item_id = random.randint(1, 10000)
+        self.client.get(f"/item?id={item_id}", name="/item")
         ...
 
-We've also declared two tasks by decorating two methods with ``@task`` and given them 
-different weights (2 and 1). When a simulated user of this type runs it'll pick one of either ``index`` 
-or ``view_post`` - with twice the chance of picking ``index`` - call that method and then pick a duration 
+We've also declared two tasks by decorating two methods with ``@task``, one of which has been given a higher weight. When a simulated user of this type runs it'll pick one of either ``hello`` 
+or ``view_item`` - with three times the chance of picking ``view_item`` - call that method and then pick a duration 
```

Comment:
I think it's worth mentioning the `weight` values here just to be extra clear.

Reply:
I added the (3)